src/App.js

```diff
@@ -17,19 +18,23 @@ import {
   Signup,
   Profile,
   SingleProduct,
+  Checkout
 } from "pages";
 import { Routes, Route } from "react-router-dom";
 import { useCart } from "context/cart-context";
 import Mockman from "mockman-js";
+import { useAddress } from "context/address-context";
 
 function App() {
   const { isModalOpen } = useCart();
+  const {isAddModalOpen} = useAddress();
```

Comment:
Suggestion - Naming for isAddModalOpen can be improved to isAddressModalOpen, which will be specific and easy to understand

Reply:
okay